cos-fleetshard-operator-debezium-it/src/test/resources/DebeziumConnectorReify.feature

```diff
@@ -70,6 +70,7 @@ Feature: Camel Connector Reify
      And the kctr has an entry at path "$.spec.pause" with value false
      And the kctr has an entry at path "$.spec.tasksMax" with value 1
      And the kctr has an entry at path "$.spec.class" with value "io.debezium.connector.postgresql.PostgresConnector"
+     And the kctr has an entry at path "$.spec.imagePullSecrets[0].name" with value "cos-pull-secret"
```

Comment:
Could you please fix the test issue related to this line @ani-sha? Here you are checking the `KafkaConnector`, but it should go to the `KafkaConnect` resource checks (some lines earlier `And kc has an entry at path...`).